src/api/routes/alerts.py

```
@@ -0,0 +1,49 @@
+async def create_alert(alert: Alert):
+    try:
+        return alert_manager.create_alert(alert)
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
+
+@router.get("/alerts/", response_model=List[Alert])
+async def get_alerts():
+    try:
+        return alert_manager.get_alerts()
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)